stages/openebs-install/tcid-iuoi02-openebs-install

```diff
@@ -9,9 +9,9 @@ sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd oep-e2e-
 
 node() {
 
-bash utils/e2e-cr jobname:tcid-iuoi02-openebs-install jobphase:Waiting
+bash utils/pooling jobname:cluster-2-health-check
 bash utils/e2e-cr jobname:tcid-iuoi02-openebs-install jobphase:Running 
-bash utils/e2e-cr jobname:tcid-ppcp01-create-cstor-pool jobphase:Waiting
+
 
 # Use cluster2 kube config
 export KUBECONFIG=~/.kube/cluster2_config
```

Comment:
We need to export it before the above CR update commands as they are running on cluster2.

Reply:
We need to do the same for `tcid-ppcp01-create-cstor-pool` test case too.

Reply:
This change exists.